embedding/cpp/flutter_app.cc

```diff
@@ -193,3 +193,92 @@ void FlutterApp::ParseEngineArgs() {
     TizenLog::Warn("Error removing file: %s", strerror(errno));
   }
 }
+
+int FlutterServiceApp::Run(int argc, char **argv) {
+  service_app_lifecycle_callback_s lifecycle_cb = {};
+  lifecycle_cb.create = [](void *data) -> bool {
+    FlutterApp *app = (FlutterApp *)data;
+    return app->OnCreate();
+  };
+  lifecycle_cb.terminate = [](void *data) {
+    FlutterApp *app = (FlutterApp *)data;
+    app->OnTerminate();
+  };
+  lifecycle_cb.app_control = [](app_control_h a, void *data) {
+    FlutterApp *app = (FlutterApp *)data;
+    app->OnAppControlReceived(a);
+  };
+
+  app_event_handler_h handler;
+  service_app_add_event_handler(&handler, APP_EVENT_LOW_MEMORY,
+                                [](app_event_info_h e, void *data) {
+                                  FlutterApp *app = (FlutterApp *)data;
+                                  app->OnLowMemory(e);
+                                },
+                                this);
+  service_app_add_event_handler(&handler, APP_EVENT_LOW_BATTERY,
+                                [](app_event_info_h e, void *data) {
+                                  FlutterApp *app = (FlutterApp *)data;
+                                  app->OnLowBattery(e);
+                                },
+                                this);
+  service_app_add_event_handler(&handler, APP_EVENT_LANGUAGE_CHANGED,
+                                [](app_event_info_h e, void *data) {
+                                  FlutterApp *app = (FlutterApp *)data;
+                                  app->OnLanguageChanged(e);
+                                },
+                                this);
+  service_app_add_event_handler(&handler, APP_EVENT_REGION_FORMAT_CHANGED,
+                                [](app_event_info_h e, void *data) {
+                                  FlutterApp *app = (FlutterApp *)data;
+                                  app->OnRegionFormatChanged(e);
+                                },
+                                this);
```

Comment:
Could you re-format this part?

```suggestion
  service_app_add_event_handler(
      &handler, APP_EVENT_LOW_MEMORY,
      [](app_event_info_h e, void *data) {
        FlutterApp *app = (FlutterApp *)data;
        app->OnLowMemory(e);
      },
      this);
  service_app_add_event_handler(
      &handler, APP_EVENT_LOW_BATTERY,
      [](app_event_info_h e, void *data) {
        FlutterApp *app = (FlutterApp *)data;
        app->OnLowBattery(e);
      },
      this);
  service_app_add_event_handler(
      &handler, APP_EVENT_LANGUAGE_CHANGED,
      [](app_event_info_h e, void *data) {
        FlutterApp *app = (FlutterApp *)data;
        app->OnLanguageChanged(e);
      },
      this);
  service_app_add_event_handler(
      &handler, APP_EVENT_REGION_FORMAT_CHANGED,
      [](app_event_info_h e, void *data) {
        FlutterApp *app = (FlutterApp *)data;
        app->OnRegionFormatChanged(e);
      },
      this);
```

Reply:
Thanks, I was using VSCode with Google style to format code, but it turned out that I had clang-format-6.0 installed. After updating it to version 10, formatting also changed.

Reply:
@pkosko Uhm, adding a code style checker for the engine and plugins repo is absolutely fine, but this repo only contains a small portion of native code and I'm not sure we need such checker for this repo. @flutter-tizen/maintainers How do you think?

Reply:
I added this page to our wiki: https://github.com/flutter-tizen/flutter-tizen/wiki/Style-guide

Please let me know if I missed something.